jpo-ode-svcs/src/test/java/us/dot/its/jpo/ode/importer/parser/BsmFileParserTest.java

```diff
@@ -98,6 +102,10 @@ public void testAll() {
          assertEquals(6, bsmFileParser.getPayloadParser().getPayloadLength());
          assertEquals(HexUtils.toHexString(expectedPayload), HexUtils.toHexString(bsmFileParser.getPayloadParser().getPayload()));
          assertEquals(expectedStep, bsmFileParser.getStep());
+         
+         ByteArrayOutputStream os = new ByteArrayOutputStream();
+         bsmFileParser.writeTo(os);
+         assertEquals(CodecUtils.toHex(buf), CodecUtils.toHex(os.toByteArray()));
       } catch (FileParserException e) {
```

Comment:
This test has a `catch` statement and also declares `throws`. We should probably stick to one or the other.

Reply:
Will remove `catch`.